test/spec/oidc/getWithRedirect.ts

```diff
@@ -86,10 +81,9 @@ describe('getWithRedirect', () => {
 
   it('redirects to the authorize endpoint', async () => {
     const { sdk, tokenParams } = testContext;
-    jest.spyOn(sdk.token.getWithRedirect, '_setLocation');
     await getWithRedirect(sdk, {});
     expect(mocked.authorize.buildAuthorizeParams).toHaveBeenCalledWith(tokenParams);
-    expect(sdk.token.getWithRedirect._setLocation).toHaveBeenCalledWith('http://fake-authorize?fake=true');
+    expect(sdk.options.setLocation).toHaveBeenCalledWith('http://fake-authorize?fake=true');
```

Comment:
similar to test with enrollAuthenticator, can you please add test with/without options.setLocation so it is clear that window.location is default behavior and options.setLocation will override